lib/HLSL/DxilValidation.cpp

```diff
@@ -2156,6 +2174,15 @@ static void ValidateDxilOperationCallInProfile(CallInst *CI,
   // Is called from a library function
   bool isLibFunc = shaderKind == DXIL::ShaderKind::Library;
 
+  // get the node launch type
+  DxilModule &DM = ValCtx.DxilMod;
+  DXIL::NodeLaunchType NLT = DXIL::NodeLaunchType::Invalid;
```

Comment:
Given the lifetime of `NLT` now, I think it needs a longer name, like `nodeLaunchType`.